websites/O/OpenStreetMap/presence.ts

```diff
@@ -0,0 +1,201 @@
+const presence = new Presence({
+		clientId: "983277228710297600"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let locationcity: string,
+	locationstate: string,
+	locationcountry: string,
+	x: string,
+	y: string,
+	editedObject: string,
+	selectedObject: boolean;
+setInterval(() => {
+	if (x && y) {
+		fetch(
+			`https://nominatim.openstreetmap.org/reverse?lat=${x}&lon=${y}&format=json`
+		)
+			.then(res => res.json())
+			.then(res => {
```

Comment:
you can keep the .then above this line but for this callback please use async/await instead